### PR TITLE
create Node.js Hello World Microservice example

### PR DESCRIPTION
**Pull request by Luca-Ferrera**

### CODE REVIEW

**Comment by fredmaggiowski** [comment]
@Luca-Ferrera there are conflicts, please resolve them.

I think that the README title shall be interpolated so it should be a simple fix.

File: default.env [hunk 1]
```diff
@@ -0,0 +1,5 @@
+USERID_HEADER_KEY=miauserid
```
Comment (fredmaggiowski) [inline default.env]: remove `mia` from these env variables.

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# %CUSTOM_PLUGIN_SERVICE_NAME%
```
Comment (fredmaggiowski) [inline docs/index.md]: This file is useless, remove it

**Review by fredmaggiowski — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## Unreleased
+
+- Inital version
```
Comment (davidebianchi) [inline CHANGELOG.md]:
```suggestion
- Initial version
```

File: scripts/update-version.sh [hunk 4]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env sh
```
Comment (davidebianchi) [inline scripts/update-version.sh:15]: We should add the copyright lines also in this file

**Review by davidebianchi — CHANGES_REQUESTED** [review]

**Review by davidebianchi — APPROVED** [review]

**Review by fredmaggiowski — APPROVED** [review]

**Pull request merged**